src/foundation/y_check_base.clas.abap

```diff
@@ -169,16 +161,11 @@ CLASS y_check_base IMPLEMENTATION.
     IF ref_scan_manager->is_scan_ok( ) = abap_false.
       RAISE EXCEPTION TYPE ycx_object_not_processed.
     ENDIF.
-
-    IF clean_code_exemption_handler->is_object_exempted( object_name = object_name object_type = object_type ) = abap_true.
-      RAISE EXCEPTION TYPE ycx_object_is_exempted.
```

Comment:
as it is not being raised here anymore, we have to remove its catch on the caller side